src/bp/ui-admin/src/Pages/Workspace/CreateUserModal.jsx

```diff
@@ -35,19 +57,78 @@ class CreateUserModal extends Component {
   }
 
   createUser = async e => {
+    const { selectedUser, selectedStrategy } = this.state
+
     e.preventDefault()
-    if (!this.isFormValid()) {
+    if (!this.isFormValid() || !selectedUser) {
       return
     }
 
-    const { data } = await api.getSecured().post('/admin/users', { email: this.state.email, role: this.state.role })
-    this.props.onUserCreated && this.props.onUserCreated(data.payload)
+    if (selectedUser['__isNew__']) {
```

Comment:
I probably missed that, where is this attribute set ?

Reply:
That's an attribute set by react-select when the entry was created (AllowCreate)